### ISSUE TITLE
[NavigationDrawer] Drawer cannot be collapsed using VoiceOver

### ISSUE THREAD

**Issue by bryanoltman** [material-components/material-components-ios#7611]
Using VoiceOver, there is no way to return an expanded navigation drawer to the collapsed state (the standard way to dismiss is to drag down). The VoiceOver cursor can only move between elements in the navigation drawer.

## Reproduction steps

1. Open the MDCDragons app
1. In settings, enable VoiceOver
1. Open the Bottom Drawer example (under the Navigation Drawer header)
1. Navigate to the hamburger menu icon
1. Tap to show the navigation drawer

## Expected behavior

The drawer is dismissible via some VoiceOver action.

## Actual behavior

There is no way to dismiss the drawer without disabling VoiceOver.

## Platform (please complete the following information)

 - Device: iPhone 8
 - OS: iOS 12.3.1
